src/phel/core.phel

```diff
@@ -477,11 +487,13 @@ Calling the and function without arguments returns true."
 
 (defn <=>
   "Alias for the spaceship PHP operator in ascending order. Returns an int."
+  {:inline (fn [a b] `(php/<=> ,a ,b))}
   [a b]
   (php/<=> a b))
```

Comment:
What about these functions where the "phel fn" is obsolete by the "inline fn"?
If they aren't used, would it be wise to remove them and write a comment like:
```newlisp
(defn <=>
  "Alias for the spaceship PHP operator in ascending order. Returns an int."
  {:inline (fn [a b] `(php/<=> ,a ,b))}
  [a b]
  # fn body removed in favor of the `:inline` function
)
```
I guess this same idea applies for all `:inline` functions without `:inline-arity`?

Reply:
The body of the function is still required, if you use the function as parameter in other function. For example

```
(apply <=> 1 [2])
(map inc [1 2 3])
# ...
```